git-to-obs-package

```diff
@@ -0,0 +1,109 @@
+#!/bin/bash
```

Comment:
hmm, having script that can be potentionally shared between more projects bundled in each git repo can cause maintenance nightmare if we use it more often